app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="comments new-comment">
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="f-label">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "Name", class: 'input' %>
+    </div>
+
+    <div class="f-label">
+      <%= f.label :bio %><br />
+      <%= f.text_field :bio, autofocus: true, autocomplete: "bio", placeholder: "Bio", class: 'input' %>
+    </div>
+
+    <div class="f-label">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'input' %>
+    </div>
+
+    <div class="f-label">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password", class: 'input' %>
+    </div>
+
+    <div class="f-label">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'input' %>
+    </div>
+
+    <div class="btn-container">
+      <%= f.submit "Sign up", class: "btn" %>
+    </div>
+  <% end %>
+
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] One of the major requirements for this project is to ensure the views in `devise` for the registration and login have been modified to fit the wireframe below. 

![image](https://github.com/rubydevi/blog-app/assets/77106962/7c8d1184-8f20-4a90-a4fe-c72a49b4fc2d)

**_Kindly modify the signup and login pages._** 👍 😎